models/order.js

```diff
@@ -4,7 +4,22 @@ const productSchema = require('./product');
 const Schema = mongoose.Schema;
 
 const orderSchema = new Schema({
-    items: [productSchema.schema],
```

Comment:
we should keep this. `items: [productSchema.schema]` encompasses all of the properties in the productSchema (like category, color, price)

we can add the total price of an order in the index function in the orders controller, instead.

but I can see how adding something for user would be helpful. I think we can add googleId as a property so that an order corresponds to the user logged in, so:

```
const orderSchema = new Schema({
    items: [productSchema.schema],
    googleId: {
        type: String,
        required: true,
    },
});
```

Reply:
Added total price to index function